source/blog/2017-10-27-webadmin-lldp.html.md

```diff
@@ -0,0 +1,72 @@
+---
+title: LLDP information in webadmin
+author: amusil
+tags: webadmin, networking
+date: 2017-10-27 09:00:00 CET
+---
+
+In oVirt 4.2 we have introduced support for LLDP protocol. 
```

Comment:
This introductory paragraph lacks an explanation of what this blog post is about, and why visitors should read on.